<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_class</class>
 <widget class="QDialog" name="add_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2,2,2,2,2,2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>添加班级信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,9">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>班级号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_classid">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入班级号(入学年份+专业代号+班级代号):如20130201</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5,4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>专业:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_major">
       <property name="placeholderText">
        <string>请输入专业全称</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5,4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>班级:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_classname">
       <property name="placeholderText">
        <string>请输入班级名称</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,5,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>人数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_memb_num">
       <property name="placeholderText">
        <string>请输入班级人数(至多2位)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>班主任:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_headteacher">
       <property name="placeholderText">
        <string>请输入班主任姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1,1,2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
